### PR TITLE
Update README and example configs

### PR DESCRIPTION
**Pull request by benraskin92**

### CODE REVIEW

File: benchmark/configs/benchmark.yml [hunk 1]
```diff
@@ -5,12 +5,12 @@ client:
       topology:
           shards: 64
           hosts:
-            - host: localhost1
+            - host: m3db_server_1
+              listenAddress: "0.0.0.0:9000"
+            - host: m3db_server_2
+              listenAddress: "0.0.0.0:9000"
```
Comment (arnikola) [inline benchmark/configs/benchmark.yml]: Should these be 9010 and 9020?

File: README.md [hunk 2]
```diff
@@ -52,6 +52,36 @@ Run Prometheus Docker image:
 
     $ docker run -p 9090:9090 -v $GOPATH/src/github.com/m3db/m3coordinator/docker/prometheus-mac.yml:/etc/prometheus/prometheus.yml quay.io/prometheus/prometheus
 
+### Running on GCP
+
+Setup GCP for m3db:
+
+    1. See m3db repo (m3db/example/README.md)
```
Comment (arnikola) [inline README.md]: Can't find where this is

Comment (benraskin92) [reply README.md]: Yeah, it's not in master yet, let me link to the PR.

File: benchmark/configs/benchmark.yml (hunk 1 above)
Comment (benraskin92) [reply benchmark/configs/benchmark.yml]: These ports are just what the user configures m3db to listen on. 9000 is the default so figured we should leave it set to those. I only had 9020 and 9030 cause i was running all of this on one host.

Comment (benraskin92) [reply benchmark/configs/benchmark.yml]: These ports are just what the user configures m3db to listen on. 9000 is the default so figured we should leave it set to those. I only had 9020 and 9030 cause i was running all of this on one host.

**Pull request merged**